sdk-api-src/content/interlockedapi/nf-interlockedapi-interlockedpushentryslist.md

```diff
@@ -1,7 +1,8 @@
 ---
 UID: NF:interlockedapi.InterlockedPushEntrySList
 title: InterlockedPushEntrySList function (interlockedapi.h)
-description: Inserts an item at the front of a singly linked list. Access to the list is synchronized on a multiprocessor system.
helpviewer_keywords: ["InterlockedPushEntrySList","InterlockedPushEntrySList function","_win32_interlockedpushentryslist","base.interlockedpushentryslist","interlockedapi/InterlockedPushEntrySList","winbase/InterlockedPushEntrySList"]
+description: Inserts an item at the front of a singly linked list. Access to the list is synchronized on a multiprocessor system.
+helpviewer_keywords: ["InterlockedPushEntrySList","InterlockedPushEntrySList function","_win32_interlockedpushentryslist","base.interlockedpushentryslist","interlockedapi/InterlockedPushEntrySList","winbase/InterlockedPushEntrySList"]

```

Comment:
That was modified by the GitHub editor, I didn't do anything

Reply:
That behaviour seems strange to me. Just decided to mention that I had the same issue in #247